### ISSUE TITLE
Add mev smoothing pool fee recipient to import view

### ISSUE THREAD

**Issue by pablomendezroyo** [dappnode/StakingBrain#253]
Add MEV smoothing pool fee recipient to the import view. There should be a toggle:
- It should be toggled by default
- If toggled, then put the mev smoothing pool fee recipient and do not allow to edit it.
- Only show the toggle if tag is "solo" staker

**Comment by dappnodedev** [comment]
I would add 2 points here:

1. Setting the Smooth fee recipient to the validators will cause those validators to be automatically subscribed to Smooth. Therefore, if they change it to another address and they propose a block they will be banned from Smooth. We must:
    1.1. Show an alert when the Smooth FR is checked in the import view that warns the users about this. Something like: "You will be automatically subscribed to Smooth once you propose a block. If you want to change the fee recipient, remember to unsubscribe here (link) before, in order to prevent this validator to be banned from Smooth"
    1.2. Make sure the "Edit Fee Recipient" feature also warns the user about this when they try to change from the Smooth fee recipient to another one

2. We might not want it to be selected by default, because if this risk-of-ban situation

**Comment by Marketen** [comment]
I agree with @dappnodedev here, we should not make this default, at least for now. If selecting Smooth as fee recipient,  we have to inform the user about the consequences of doing so (link to docs, autosubscribe, etc).

User shouldnt be able to edit fee recipient if toggle is active, like @pablomendezroyo said. We could add a "I have unsubscribed from smooth" button to enable editing fee recipient of  that validator again.
